Terms of Reference.md

```diff
@@ -0,0 +1,140 @@
+# Terms of Reference
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Project Scope](#project-scope)
+- [Purpose of the Technical Committee](#purpose-of-the-technical-committee)
+- [Membership Composition](#membership-composition)
+- [Project Schedule](#project-schedule)
+- [Meeting Schedule](#meeting-schedule)
+- [Consensus Building](#consensus-building)
+- [Code of Conduct](#code-of-conduct)
+
+## Overview
+
+This Terms of Reference (ToR) defines the goals of the Trust Meter Technical
+Specification, how it differs or relates to other specifications regarding
+Artificial Intelligence (AI) and Trust, and the objectives of the Technical
+Committee, outlining the roles and responsibilities of those involved.
+
+## Project Scope
+
+The Trust Meter Technical Specification addresses statistical discrimination
+against data outliers and small minorities in mechanized statistical reasoning
+in AI decision tools. The Trust Meter Technical Specification is intended to
+alert AI operators when the scenario, group or individual about whom the
+decision is made is out-of-distribution relative to the training set the model
+is trained on, meaning the AI should not be trusted to make decisions in this
+context.
+
+## Purpose of the Committee
```

Comment:
I wonder if this needs to be mention information about the drafting/writing of the Technical Specification.

Reply:
Based on last week's meeting, I added a sub-section of the "Meeting Schedule" named "Meeting Methodology and Collaborative Processes".   The new sub-section says that the committee needs to decide  for themselves how they want to meet, how they want to "... draft the specifications, including tracking versions of it".  I also added a set of links from the "Purpose of the Committee" dealing with administrative tasks and included a link to the new "Meeting Methodology and Collaborative Processes" sub-section.

So, drafting/writing the specification is mentioned, but I'm not sure if it's explicit enough.